astropy/stats/funcs.py

```diff
@@ -216,39 +216,40 @@ def binom_conf_interval(k, n, confidence_level=0.68269, interval='wilson'):
     --------
     Integer inputs return an array with shape (2,):
 
-    >>> binom_conf_interval(4, 5, interval='wilson')
+    >>> binom_conf_interval(4, 5, interval='wilson')  # doctest: +FLOAT_CMP
     array([0.57921724, 0.92078259])
 
     Arrays of arbitrary dimension are supported. The Wilson and Jeffreys
     intervals give similar results, even for small k, N:
 
-    >>> binom_conf_interval([0, 1, 2, 5], 5, interval='wilson')
+    >>> result = binom_conf_interval([0, 1, 2, 5], 5, interval='wilson')
+    >>> print(np.array_repr(result, precision=8, suppress_small=True))  # doctest: +FLOAT_CMP
```

Comment:
I don't know how to make this pretty...

Reply:
Sigh. Really this is a bit of a bug in `FLOAT_CMP`, though in fairness I wouldn't know how it can know when small and zero are the same...  But I think we shouldn't go this route -- the disadvantage to the reader of the docstring is larger than the advantage of testing that this works... How about just skipping the 0 in the example (and removing the first entry of each row in the result)?

Reply:
Like this?
```python
    >>> binom_conf_interval([0, 1, 2, 5], 5, interval='wilson')  # doctest: +FLOAT_CMP +ELLIPSIS
    array([[..., 0.07921741, 0.21597328, 0.83333304],
           [..., 0.42078276, 0.61736012, 1.        ]])
```

I did try to replace `0` with `...` but that didn't work, but I also didn't try to remove the first entry in the second row.

Reply:
I think it would be nicer to remove the 0 in the input and remove both now dotted entries.  Really, `0, 1, 2, 5` was just taken as an example, so one might as well reduce it to `[1, 2, 5]`.

Reply:
Basically, I think that the docs should remain aimed at someone *reading* them, not at testing. So, also completely happy to just add `doctest: +SKIP` and be done with it!!

Reply:
Good idea! Probably the best workaround between setting print options and skipping it outright.